journal-of-agricultural-and-applied-economics.csl

```diff
@@ -0,0 +1,664 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" et-al-min="3" et-al-use-first="1" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Journal of Agricultural and Applied Economics</title>
+    <title-short>JAAE</title-short>
+    <id>http://www.zotero.org/styles/journal-of-agricultural-and-applied-economics</id>
+    <link href="http://www.zotero.org/styles/journal-of-agricultural-and-applied-economics" rel="self"/>
+    <link href="https://www.saea.org/wp-content/uploads/2013/08/reference-style-sheet.pdf" rel="documentation"/>
```

Comment:
@jmthomp, do you know where you found these instructions at https://www.saea.org/wp-content/uploads/2013/08/reference-style-sheet.pdf?

When I went through the SAEA website, http://www.saea.org/jaae/manuscript-submission-guidelines/ directed me to https://www.cambridge.org/core/journals/journal-of-agricultural-and-applied-economics/information/instructions-contributors, which seems to list the same examples at the end of the page. That's probably a better documentation link.

Reply:
The style sheet, is on http://www.saea.org/jaae/manuscript-submission-guidelines/ At the top there is a menu called JAAE --> then Reference Style Sheet. That was the pdf. The same references are in the contributors. As frustrating as it is these are the guidelines for this journal. So no other information on reference style other than those examples.

Reply:
Whatever you feel is best. It's all the same information. :)Thanks for your help.